app/controllers/main/routes.py

```diff
@@ -113,3 +114,9 @@ def RemoveRSVP():
 
     flash("Successfully unregistered for event!", "success")
     return redirect(url_for("admin.editEvent", eventId=eventId, program=program))
+@main_bp.route('/contributors',methods = ['GET'])
+def contributors():
+    # contribs = app.config['contributors']
+    return render_template("/contributors.html"
+           # cfg = contribs
```

Comment:
don't just leave it commented out. It's not useful anymore, and if someone really wants to see what it used to be they can look at the history of the file.